t/10-virtio_terminal.t

```diff
@@ -15,6 +15,8 @@ use consoles::virtio_terminal;
 use testapi;
 use bmwqemu;
 
+my $pipe_data_written;
+sub wait_till_pipe_data_written() { sleep 1 while (!$pipe_data_written) }
```

Comment:
I think Perl let's you write this more nicely:
```suggestion
sub wait_till_pipe_data_written() { sleep 1 until $pipe_data_written }
```

---

Maybe one second is also a bit long here? Or will the signal handler interrupt the sleep anyways? In this case a comment like `  # sleep interrupted by SIGUSR2` would make sense.

Reply:
It actually doesn't matter that much. 
Sleep will interrupt if the SIGUSR2 will arise and if it was already emitted, then we do not sleep at all. Only a very slow chance left to hit the `sleep 1` unnecessary.
But sure, why not to wait 0.5.

Reply:
If the sleep is interrupted anyways then the interval shouldn't matter much (except the interrupt happens between the evaluation of the loop condition and the execution of the loop body).